pom.xml

```diff
@@ -32,6 +32,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <lombok.version>1.18.12</lombok.version>
     <spring.version>5.2.7.RELEASE</spring.version>
+    <mod-audit-client.version>0.0.5-SNAPSHOT</mod-audit-client.version>
```

Comment:
Yes, Marc, it's a good question too. We haven't functionality that we can release right now for having separate release. I think we just need to sync-up releases of mod-audit and mod-circulation. I'm going to change this version during releasing process. What do you think on this?

Reply:
> We haven't functionality that we can release right now for having separate release. 

I don't really understand that statement. Why couldn't the functionality that mod-circulation is using right now be released?

I've had similar conversations with the FoliJet team recently as well, and I accepted the use of snapshot versions because they felt strongly about not releasing their libraries yet. We can do that in this case as well :-/ 

> I think we just need to sync-up releases of mod-audit and mod-circulation. I'm going to change this version during releasing process. What do you think on this?

My concern with this approach is that the folks responsible for mod-circulation (me) are not the same folks responsible for releasing mod-audit, meaning this introduces additional coordination effort required during the release processes.